packages/react-composites/src/composites/MeetingComposite/NotificationIcon.tsx

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { Icon, Stack } from '@fluentui/react';
+import { NotificationIconStyles, NotificationTextStyles } from './styles/MeetingCompositeStyles';
+
+/**
+ * @private
+ */
+export type NotificationIconProps = {
+  numberOfMessages: number;
+};
+/**
+ * @private
+ */
+export const NotificationIcon = (props: NotificationIconProps): JSX.Element => {
+  const { numberOfMessages } = props;
+  return (
+    <Stack horizontalAlign="center" verticalAlign="center" style={{ position: 'absolute', top: 0, right: '0.78rem' }}>
```

Comment:
fluent recomendation is to use the `styles` prop instead of `style` (styles gets merged into a single classname that can be better reused, cached etc., style will directly put the css into the element in the html file)
https://github.com/microsoft/fluentui/wiki/Component-Styling

so this will become something like:
```suggestion
    <Stack horizontalAlign="center" verticalAlign="center" style={notificationIconContainerStyles}>
```
then at the bottom of the page as a *constant* (being a constant outside the render scope avoids creating a new object every render):
```tsx
const notificationIconContainerStyles = { position: 'absolute', top: 0, right: '0.78rem' }
```

Reply:
Curious: I couldn't find any pros and cons of using `styles` over inline `style` on the link you shared.